src/pages/api/student/[email]/index.ts

```diff
@@ -0,0 +1,47 @@
+import { updatePasswordByEmail } from "@/backend/actions/student";
```

Comment:
Same thing here, just for organizing purposes to maybe add what's a 3rd Party Import and what's a Local Import